### PR TITLE
Feat/DJDP 1132/define testing strategy

### PR DESCRIPTION
**Pull request by germfy**
### Description
This PR includes:
Github actions workflows for Dev and main branch to review code quality and integrate the package into an artifact 

### Tickets
[DJDP-1132](https://wizeline.atlassian.net/browse/DJDP-1132)

### CODE REVIEW

File: .github/workflows/dev_branch_linting.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: |
+          python -m pip install --upgrade pip
+      
+      - name: Install required libraries
+        run : |
+          python -m pip install pylama eradicate pylint
+
+      - name: Pylama analysis
+        run :  |
+          pylama -l pylint, pycodestyle, pydocstyle ./
```
Comment (vic3r) [inline .github/workflows/dev_branch_linting.yml:32]: I think we will need to create a ticket, for configuring Pylama settings in a file

File: .github/workflows/main_branch_linting_package.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        run: |
+          python -m pip install --upgrade pip
+      
+      - name: Install required libraries
+        run : |
+          python -m pip install pylama eradicate pylint
+
+      - name: Pylama analysis
+        run :  |
+          pylama -l pylint, pycodestyle, pydocstyle ./
```
Comment (vic3r) [inline .github/workflows/main_branch_linting_package.yml:32]: Is is possible to reuse the whole linting job previously defined?

**Review by vic3r — APPROVED** [review]
👍

File: .github/workflows/main_branch_linting_package.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+name: "RDL CI/CD workflow for Main branch"
+
+on:
+  push:
+    branches: 
+     - main
```
Comment (cerritows) [inline .github/workflows/main_branch_linting_package.yml:6]: Our main branch is still called `master` not `main` 😞

**Review by cerritows — COMMENTED** [review]
👁️ .👁️

**Review by cerritows — APPROVED** [review]
🚀

**Pull request merged**